<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DienstnehmerDetailForm</class>
 <widget class="QDialog" name="DienstnehmerDetailForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dienstehmer Details</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_dienstnehmer">
     <property name="title">
      <string>Dienstnehmer</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Numme&amp;r</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_nummer</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_nummer">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>&amp;SVNr.</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_svnr</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_svnr">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>2</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_color">
            <property name="text">
             <string>&amp;Farbe</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&amp;Nachname</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_nachname</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_nachname">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>2</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>&amp;Vorname</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_vorname</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_vorname">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>2</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Max. &amp;Lohn</string>
            </property>
            <property name="buddy">
             <cstring>doubleSpinBox_maxLohn</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_maxLohn">
            <property name="maximum">
             <double>99999.990000000005239</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>&amp;Bereich</string>
            </property>
            <property name="buddy">
             <cstring>comboBox_bereich</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_bereich"/>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>&amp;Gehalt</string>
            </property>
            <property name="buddy">
             <cstring>comboBox_gehalt</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_gehalt"/>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Ereignisse</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_newEreignis">
          <property name="text">
           <string>&amp;Hinzufügen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_deleteEreignis">
          <property name="text">
           <string>&amp;Löschen</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tableView_ereignisse"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_nummer</tabstop>
  <tabstop>lineEdit_svnr</tabstop>
  <tabstop>pushButton_color</tabstop>
  <tabstop>lineEdit_nachname</tabstop>
  <tabstop>lineEdit_vorname</tabstop>
  <tabstop>doubleSpinBox_maxLohn</tabstop>
  <tabstop>comboBox_bereich</tabstop>
  <tabstop>comboBox_gehalt</tabstop>
  <tabstop>pushButton_newEreignis</tabstop>
  <tabstop>pushButton_deleteEreignis</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>tableView_ereignisse</tabstop>
 </tabstops>
 <resources>
  <include location="../../lagerManager.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DienstnehmerDetailForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DienstnehmerDetailForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
